### PR TITLE
Feature/dwr 301 home page

### PR DESCRIPTION
**Pull request by bgsavage**
Added button links and dynamic html content to the translation.

### CODE REVIEW

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 1]
```diff
@@ -212,8 +216,6 @@
         }
       }
     },
-    "user-guide": "User Guide",
-    "interpretive-guide": "Interpretive Guide",
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: Gave these orphans a nice new home.

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 2]
```diff
@@ -345,5 +347,8 @@
...
...lue-dark h3 mb-md\">Summary Reports</h2><div class=\"summary-reports-container mb-md\"><p><a href=\"#\" class=\"h5 blue\">Achievement Level Report</a><br />Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque et delectus cumque at illo minima accusamus.</p><p><a href=\"#\" class=\"h5 blue\">Achievement Standard Report</a><br />Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque et delectus cumque at illo minima accusamus.</p><p><a href=\"#\" class=\"h5 blue\">Longitudinal Data Report</a><br />Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque et delectus cumque at illo minima accusamus.</p></div><!-- Member Reporting Resources --><h2 class=\"blue-dark h3 mb-md\">Member Reporting Resources</h2><div class=\"member-reporting-resources-container\"><p><a href=\"#\" class=\"h5 blue\">Member-provided Resources</a><br />Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p><p><a href=\"#\" class=\"h5 blue\">How to Establish PII Permissions</a><br />Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p><p><a href=\"#\" class=\"h5 blue\">How to Create Student Groups</a><br />Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p></div>"
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: I wish there was a way to format this better in JSON for readability.

File: webapp/src/main/webapp/src/app/home/home.component.html [hunk 3]
```diff
@@ -40,33 +40,13 @@ <h1 class="blue-dark h1 mb-md">{{'labels.homepage.title' | translate}}</h1>
...
-        Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
-        <p><a href="#" class="h5 blue">How to Establish PII Permissions</a><br />
-        Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
-        <p><a href="#" class="h5 blue">How to Create Student Groups</a><br />
-        Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
+        <p><a href="#" class="btn btn-success">{{'labels.navigation.interpretive-guide' | translate}}</a></p>
       </div>
 
+      <!-- Summary News Dynamic Content -->
+      <div [innerHTML]="systemNewsHtml"></div>
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/app/home/home.component.html]: The default behavior is the most secure where angular will strip what it deems as unsafe html.  For what we are trying to accomplish, this should be fine.

File: webapp/src/main/webapp/src/assets/i18n/en.json (hunk 2 above)
Comment (wtritch) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: Once this is "for reals" it won't have HTML in it, right?

Comment (esotrope) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: Nope the idea here is to support html

**Pull request merged**